typescript/src/events/guilds/members/guildMemberUpdateNicknameNotify.ts

```diff
@@ -11,20 +10,16 @@ import type { TFunction } from 'i18next';
 @ApplyOptions<EventOptions>({ event: Events.GuildMemberUpdate })
 export class UserEvent extends Event {
 	public async run(previous: GuildMember, next: GuildMember) {
-		const [enabled, t] = await next.guild.readSettings((settings) => [
-			settings[GuildSettings.Events.MemberNickNameUpdate],
-			settings.getLanguage()
-		]);
-		// Retrieve whether or not nickname logs should be sent from Guild Settings and
-		// whether or not the nicknames are identical.
-		if (!enabled) return;
+		const key = GuildSettings.Channels.Logs.MemberNickNameUpdate;
+		const [channelId, t] = await next.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

Comment:
```suggestion
		const [logChannelId, t] = await next.guild.readSettings((settings) => [settings[key], settings.getLanguage()]);
```

And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.